tracing/propagation/opencensus/propagation.go

```diff
@@ -0,0 +1,96 @@
+package opencensus
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+	"go.opencensus.io/trace"
+)
+
+const (
+	TraceIdKey      = "go-shuttle/traceId"
+	SpanIdKey       = "go-shuttle/spanId"
+	TraceSampledKey = "go-shuttle/traceSampled"
+)
+
+func SpanContextFromMessage(msg *servicebus.Message) (trace.SpanContext, bool) {
+	sc := trace.SpanContext{}
+
+	// traceID
+	traceID, ok := getDecodedBytes(msg.UserProperties, TraceIdKey)
+	if !ok {
+		return trace.SpanContext{}, false
+	}
+	copy(sc.TraceID[:], traceID)
+
+	// spanID
+	spanID, ok := getDecodedBytes(msg.UserProperties, SpanIdKey)
+	if !ok {
+		return trace.SpanContext{}, false
+	}
+	copy(sc.SpanID[:], spanID)
```

Comment:
why copy? why not just assign sc.TraceID/SpanID the result from getDecoded bytes?